test/Microsoft.AspNet.Mvc.FunctionalTests/ModelBindingTest.cs

```diff
@@ -2067,6 +2067,23 @@ public class ModelBindingTest
             Assert.False(result[1].Checked);
         }
 
+        [Fact]
+        public async Task TryUpdateModel_InvalidModelAsParameter_FixOnServer_ReturnsEmptyResponseBody()
```

Comment:
I not able to understand what the test does. what does ```FixOnServer_ReturnsEmptyResponseBody``` mean?
Should it be 
```TryUpdateModel_ClearsModelStateEntries``` instead?